.eslintrc

```diff
@@ -1,22 +1,17 @@
 {
   "parser": "babel-eslint",
-  "extends": [
-    "standard",
-    "standard-react"
-  ],
+  "extends": ["airbnb"],
```

Comment:
still, there is a devDependency: "eslint-config-standard" and you just changed the name didn't configure it